### ISSUE TITLE
Support older versions of .NET Framework

### ISSUE THREAD

**Issue by altso** [Haacked/Scientist.net#49]
As far as I can see from nuget package, Scientist.net currently targets net452. Is it possible to add support for older versions of .NET Framework? If so, what would be the lowest version supported?

**Comment by davezych** [comment]
We are on .NET Core, and are currently targeting `dotnet5.2` which is a platform standard, e.g. it allows us to target multiple platforms at once. [This link has a lot of information](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md), but [this is the useful part.](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#mapping-the-net-platform-standard-to-platforms). 

(names are changing which is why the `dotnet5.x` monikors don't show up, but they are direct-mapped to the `netstandard` monikors. In our case `dotnet5.2` == `netstandard1.1`)

If you look at the table, `netstandard1.1` targets a lot of things, one of which is `.NET 4.5`. 

Basically, this is a long way of saying we support `.NET 4.5`. I don't think we'll target anything lower than 4.5.

The latest packages on Nuget were published ~2 months ago, and haven't been updated to the latest bits since we have made the change to target 4.5.

**Comment by altso** [comment]
@davezych thank you for a prompt but detailed response. Any plans to update the nuget package any time soon?

**Comment by davezych** [comment]
Hopefully soon but I'm not sure. Perhaps @Haacked can shed some light.

**Comment by Haacked** [comment]
I'm ready to publish soon. @davezych do we automatically build the NuGet package now? I've been on vacation and now all my OSS projects blur together and I can't remember what we settled on here. Call it vacation brain. :stuck_out_tongue: :palm_tree:

**Comment by davezych** [comment]
@Haacked welcome back!

Yep, the nuget packages are published as artifacts in appveyor: https://ci.appveyor.com/project/Haacked15676/scientist-net/build/1.0.122/artifacts

**Comment by Haacked** [comment]
Ok, I pushed them!

**Comment by Haacked** [comment]
Please try it out. I'll close this issue for now. Comment here if there's still problems.

**Comment by altso** [comment]
@Haacked thank you!

**Comment by davezych** [comment]
Hey @Haacked do you know what build you pushed to nuget? I'm testing things in .NET 4.5 to make sure it works properly, and the context stuff doesn't look like it's there. Everything else works though.

**Comment by Haacked** [comment]
@davezych I thought it was the latest. Context is merged to master, right?

**Comment by Haacked** [comment]
@davezych in that other thread, you sent me a link to the AppVeyor build artifacts. I published whatever was in there.

**Comment by davezych** [comment]
That was the latest build from the context pull request branch, not from master. My bad, I added the link as an example and didn't specify that.

Curiously enough, though, that *should* include the context things... that commit is the one you merged into master, so I'm not really sure why the context stuff isn't in the nuget package.

The latest build artifacts are here: https://ci.appveyor.com/project/Haacked15676/scientist-net/build/artifacts, but we'll need to increment the beta # prior to another nuget release. 
I just manually pulled down and ref'd this build and confirmed context things are available.

**Comment by Haacked** [comment]
Doh! I must have had vacation brain. Would you do me a favor? Create a PR that just revs the version and release notes (if necessary) and I'll publish the NuGet artifacts from that.

I am flying to Peru next week to give a keynote and session and I kind of need to prepare for them so I've been kind of swamped. :grimacing:
